backend/src/common/constants/pagination.constants.ts

```diff
@@ -0,0 +1,4 @@
+const DEFAULT_PAGE = 1;
+const DEFAULT_COUNT = 10;
```

Comment:
set of constants is always an enum
move it to the shared folder
we will reuse it